nncf/experimental/onnx/helper.py

```diff
@@ -0,0 +1,131 @@
+"""
```

Comment:
Again, the name of the file does not reflect the content. My suggestion is: "ImageNetDataLoader" can be moved to a separate file, like "imagenet_dataloader.py" and the rest of the content looks like a "util.py".

Reply:
I've created the new folder - `nncf/experimental/onnx/dataloaders/imagenet_dataloader.py`
Also, renamed `helper.py` to `utils.py`

Reply:
You keep on adding the `utils.py` files. There is hardly any good reason to have a "utils" file anywhere in a proper OOOP code. Group these functions in a well-defined object or objects instead, and have a file that has a name consistent with the defined objects.

Reply:
There are two functions that modify the ONNX model once at the begging of the pipeline.  I can create ONNModel class for that in the file named onnx_model.py and put in onnx/graph/onnx_model.py.